### ISSUE TITLE
call lua function in actions

### ISSUE THREAD

**Issue by ViRu-ThE-ViRuS** [gfanto/fzf-lsp.nvim#20]
I was wondering if its possible to add functionality where we can set a lua function to be called instead of a vim cmd using `g.fzf_lsp_actions`? 

I have a custom qf list populating method in lua, which I use along with `fzf.vim` and a couple of custom LSP handlers. It would be great if I can get the same behavior with the qflist made by `fzf-lsp`.

**Comment by gfanto** [comment]
Right now with the current setup is not possible, but it should be easy to add.
Let me check if i understand correctly, you want to do something like:
viml version
```vim
let g:fzf_lsp_action = {
  'ctrl-t': call luaeval("my_custom_action_fn")
}
```

lua version
```lua
vim.g.fzf_lsp_action = {
  "ctrl-t" = my_custom_fn,
}
```

with `my_custom_fn` be like:
```lua
local function my_custom_fn(fzf_lines)  -- fzf_lines is lua table containing all the selected lines
  -- ...
end
```

Is this what are you looking for?

**Comment by ViRu-ThE-ViRuS** [comment]
yes I think it would be a great addition to this plugin, and allow its users to get more creative with handling the selected entries from FZF.
